forwarder.go

```diff
@@ -219,15 +246,16 @@ func init() {
 		MaxIdleConnsPerHost: workers,
 	}
 	client = &http.Client{Transport: transport}
+	timestampRegex, _ = regexp.Compile("([0-9]+)-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])[Tt]([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9]|60)(.[0-9]+)?(([Zz])|([+|-]([01][0-9]|2[0-3]):[0-5][0-9]))")
```

Comment:
`MustCompile` would panic rather than throw an error that gets discarded. As this is a static value, we should be confident the panic will never happen.